src/App.css

```diff
@@ -31,3 +31,13 @@ button#choose-file svg {
 .cdc-react .usa-alert .usa-alert__text {
   font-family: sans-serif !important;
 }
+
```

Comment:
Can we move this table related css into the Table.module.css file, and import that directly? Keep the global styles a bit cleaner?